app.rb

```diff
@@ -0,0 +1,137 @@
+require './person'
+require './student'
+require './teacher'
+require './books'
+require './rental'
+
+class App
+  def run
+    @persons = []
+    @books = []
+    puts "\nWelcome to the School Library App!"
+    display_menu
+  end
+
+  def display_menu
+    puts "\nPlease choose an option by entering a number:"
+    puts "\n1 - List all books"
+    puts '2 - List all people'
```

Comment:
- [ ] When listing all people it shows a wrong name in student and a wrong name and specialization from the teacher.

![Screen Shot 2022-03-11 at 11 21 02](https://user-images.githubusercontent.com/67211919/157916390-8425d466-8c60-42c9-a51c-1ca5a2ce44b1.png)